src/controllers/IndexController.php

```diff
@@ -333,15 +333,44 @@ class="fb-main page--team-registration full-height fb-scroll">
             </fieldset>
             <div class="fb-choose-emblem">
               <h6>{tr('Choose an Emblem')}</h6>
-              <div class="emblem-carousel">{$logos_section}</div>
+              <h6>
+                <a href="#" id="custom-emblem-link">or upload your own</a>
+              </h6>
+              <div class="custom-emblem">
+                <input
+                  autocomplete="off"
+                  name="custom-emblem"
+                  id="custom-emblem-input"
+                  type="file"
+                  accept="image/*"
+                />
+                <img
+                  id="custom-emblem-preview"
+                  src=""
+                  height={62}
+                  width={80}>
+                </img>
+              </div>
+              <div class="emblem-carousel">
+                <div id="custom-emblem-carousel-notice">
+                  <div class="center-wrapper">
+                    <h6>
+                      <a href="#" id="custom-emblem-clear-link">
+                        Clear your custom emblem to use a default emblem
+                      </a>
+                    </h6>
+                  </div>
+                </div>
+                {$logos_section}
+              </div>
             </div>
             <div class="form-el--actions fb-container container--small">
               <p>
                 <button
                   id="register_button"
                   class="fb-cta cta--yellow"
                   type="button">
-                  {tr('Sign Up')}
+                  Sign Up
```

Comment:
Good catch! Will go ahead and add the translation wrapper for the various strings that were missed.